config/jobs/gardener-extension-shoot-rsyslog-relp/gardener-extension-shoot-rsyslog-relp-integration-tests.yaml

```diff
@@ -0,0 +1,59 @@
+presubmits:
+  gardener/gardener-extension-shoot-rsyslog-relp:
+  - name: pull-gardener-extensioon-shoot-rsyslog-relp-integration
+    cluster: gardener-prow-build
+    always_run: true
+    skip_branches:
+    - release-v\d+.\d+ # don't run on release branches for now (add a job per branch later)
+    decorate: true
+    decoration_config:
+      timeout: 20m
+      grace_period: 10m
+    annotations:
+      description: Runs integration tests for Gardener extension shoot-rsyslog-relp developments in pull requests
+      fork-per-release: "true"
+    spec:
+      containers:
+      - name: test-integration
+        image: eu.gcr.io/gardener-project/ci-infra/golang-test:v20230810-2a54385-1.21
+        command:
+        - make
+        args:
+        - test-integration
+        resources:
+          limits:
+            memory: 16Gi
+          requests:
+            cpu: 5
+            memory: 8Gi
```

Comment:
Did you measure/estimate these requests or is this just because it is a copy form `gardener/gardener`?
Same for the other resource requests.